core/server/worker/src/main/java/alluxio/worker/block/meta/StorageDir.java

```diff
@@ -109,11 +111,11 @@ public static StorageDir newStorageDir(StorageTier tier, int dirIndex, long capa
    * @throws WorkerOutOfSpaceException when metadata can not be added due to limited left space
    */
   private void initializeMeta() throws BlockAlreadyExistsException, IOException,
-      WorkerOutOfSpaceException {
+      WorkerOutOfSpaceException, InvalidPathException {
     // Create the storage directory path
     boolean isDirectoryNewlyCreated = FileUtils.createStorageDirPath(mDirPath,
         ServerConfiguration.get(PropertyKey.WORKER_DATA_FOLDER_PERMISSIONS));
-
+    String tmpDir = ServerConfiguration.get(PropertyKey.WORKER_DATA_TMP_FOLDER).split("/")[0];
```

Comment:
can we use a method from `PathUtils` instead?

Reply:
Done as per your suggestion.